js/details.js

```diff
@@ -64,33 +65,11 @@
           }, this);
         }, this);
 
-        var actions = [
-          {
-            title: 'Install this theme',
-            action: 'install'
-          },
-          {
-            title: 'Fork this theme',
-            action: 'fork'
-          },
-          {
-            title: 'Remove this theme',
-            action: 'remove',
-            css: 'destructive'
-          }
-        ];
-        actions.forEach(function(params) {
-          var link = document.createElement('a');
-          link.classList.add('action');
-          link.dataset.action = params.action;
-          if (params.css) {
-            link.classList.add(params.css);
-          }
-          var title = document.createElement('h3');
-          title.textContent = params.title;
-          link.appendChild(title);
-          this.list.appendChild(link);
-        }, this);
+        if (!currentTheme.manifestURL) {
+          this.enableButton.classList.add('disabled');
+        } else {
+          this.enableButton.classList.remove('disabled');
+        }
```

Comment:
this part of the feature is not working for me :/

1. I think we want the button to be disabled when `currentTheme.manifestURL` is truthy, not the reverse
2. We need to refresh this class at the end of the `installTheme` method (when we refresh `currentTheme` who should now have a `manifestURL`)
3. Don't know if we have a `.disabled` style for the button but we might need to add one (maybe just an `opacity:0.7`) and make sure it has a `pointer-events: none`
4. Finally, we don't have a spinner/loader while the theme zip is being generated so to prevent multiple generation from being triggered by a "double tap" we probably want to set the `.disabled` class as soon as we start the `installTheme` process